assets/script.js

```diff
@@ -196,13 +200,17 @@ function getArtistInfo(userInput) {
             } else {
               // Appending the album
               $("#albums>ul").append(
-                '<li><a id="albums" class="waves-effect waves-light modal-trigger" href="#album-modal"><img src="' +
+                '<li><a id="albums" class="image waves-effect waves-light modal-trigger" href="#album-modal"><img class="materialboxed" src="' +
                   albumImage +
-                  'alt="' +
+                  '" alt="' +
                   albumName +
-                  "/></a>" +
+                  '" title=" ' + albumName + ' by ' + response.topalbums["@attr"].artist +
```

Comment:
Descriptive attribute names. What is this the title of? Remember that people looking at this probably won't have been working on it for a week and a half and won't have a clue what you're on about

Reply:
Also, by concatting this in this way it will be extremely difficult to actually use this attribute at all. need to have two: one for artist and one for album